### DISCUSSION TITLE
QUESTION: Adding selected component style to a block

### DISCUSSION THREAD

**Discussion by aryanchopra** [GrapesJS/grapesjs#3636] [Q&A]
So I'm trying to fetch the selected component and make a block out of it, however, I'm struggling with adding the css of the same. my approach looks like this currently:
`editor.BlockManager.add(widgetname, {
            label: widget name,            
            content: editor.getSelectedAll(),
            category: panelname,
            attributes: {
              title: widgetname,
              class: iconclass              
            },
          });`
Any leads would be helpful :)

**Comment by GaikwadShubham** [comment]
Hi @aryanchopra, I would like to help you solve this problem. Can you make a demo on codesandbox something like [this](https://codesandbox.io/embed/frosty-payne-3fhd7?fontsize=14&hidenavigation=1&theme=dark)
